tests/test_request.py

```diff
@@ -7,6 +7,25 @@
 
 class TestRequest:
 
+    def test_get_path(self, bobtail_app, environ):
+        class Images:
+            def get(self, req, res):
+                pass
+
+        routes = [(Images(), "/images/{id:int}")]
+
+        app = bobtail_app(routes=routes)
+
+        env = environ(path="/images/1")
+        data = app(env, lambda s, r: None)
+        req_path = app.request.get_path()
+        assert req_path == "/images/1"
+
+        env = environ(path="/images?q=monalisa")
+        data = app(env, lambda s, r: None)
+        req_path = app.request.get_path()
+        assert req_path == "/images?q=monalisa"
```

Comment:
@joegasewicz should the path contain query params? I can return only path after parsing if that's the expectation.

Reply:
That's perfect, thank you